stable/percona-xtradb-cluster/files/entrypoint.sh

```diff
@@ -9,6 +9,9 @@ fi
 
 ipaddr=$(hostname -i | awk ' { print $1 } ')
 hostname=$(hostname)
+# SHORT_CLUSTER_NAME exists in order to fulfill `wsrep_cluster_name` variable max size.
+# https://www.percona.com/doc/percona-xtradb-cluster/LATEST/wsrep-system-index.html#wsrep_cluster_name
+SHORT_CLUSTER_NAME=${CLUSTER_NAME/*cluster-/cluster-}
```

Comment:
I think I'd rather see helm create the short cluster name and then pass it in as an environment variable or similar.  That way anything else that may need to know the wsrep cluster name can use that as well rather than relying on the entrypoint script to set it.

Reply:
hey @paulczar thanks for the feedback. I'm changing it. Thanks

Reply:
@paulczar what do you think about using just the `Release.Name` for the name of the cluster?